src/tr2/config_map.def

```diff
@@ -10,6 +10,7 @@ CFG_BOOL(g_Config, visuals.fix_item_rots, true)
 CFG_ENUM(g_Config, rendering.screenshot_format, SCREENSHOT_FORMAT_JPEG, SCREENSHOT_FORMAT)
 CFG_ENUM(g_Config, rendering.render_mode, RM_HARDWARE, RENDER_MODE)
 CFG_ENUM(g_Config, rendering.aspect_mode, AM_ANY, ASPECT_MODE)
+CFG_ENUM(g_Config, rendering.lighting_contrast, LIGHTING_CONTRAST_HIGH, LIGHTING_CONTRAST)
```

Comment:
It may just be my setup but the "low" option is what feels closest to OG PC for me. Should low be the default? Medium and high feel quite dark.